strawberry/starlite/handlers/graphql_transport_ws_handler.py

```diff
@@ -0,0 +1,63 @@
+from datetime import timedelta
+from typing import Any
+
+from starlite import WebSocket
+from starlite.exceptions import WebSocketDisconnect
+from strawberry.schema import BaseSchema
+from strawberry.subscriptions import GRAPHQL_TRANSPORT_WS_PROTOCOL
+from strawberry.subscriptions.protocols.graphql_transport_ws.handlers import (
+    BaseGraphQLTransportWSHandler,
+)
+
+try:
+    from msgspec import MsgspecError as JSONDecodeError
+except ModuleNotFoundError:
```

Comment:
It would be an ImportError, not ModuleNotFoundError, but it is bundled with Starlite, so you should simply reraise the exception telling the user starlite is not installed